addons/stock/models/stock_orderpoint.py

```diff
@@ -209,8 +209,11 @@ def write(self, vals):
     def action_open_orderpoints(self):
         return self._get_orderpoint_action()
 
-    def action_replenish(self):
-        self._procure_orderpoint_confirm(company_id=self.env.company)
+    def action_replenish(self, raise_user_error=True):
+        # Force to recompute because if, for example, a RFQ cancelled after last
+        # compute, the qty_forecast won't be correct.
+        self._compute_qty()
+        self._procure_orderpoint_confirm(company_id=self.env.company, raise_user_error=raise_user_error)
```

Comment:
@amoyaux will make a different patch (he thinks a depends is missing)

Reply:
removed